README.md

```diff
@@ -33,7 +33,8 @@ To add library in your gradle project follow the steps below:
 2. Add the dependency
    ```groovy
    dependencies {
-       implementation 'com.github.openMF:mifos-android-sdk-arch:1.0.0'
+       def sdk_Version = "replace with the version number"
```

Comment:
Let's put a version number here if we want to release after this PR?

Reply:
I thought that for each release we have to update version in readme so why not generalise this, that's why I have kept like this. Let update the PR.

Reply:
Done, please check. Thanks :)